docs/design/fabric-v2+/fpc-registration.puml

```diff
@@ -40,7 +40,7 @@ end note
 
 Admin -> Admin: Attestation_Params <- <.e.g, SPID, sig-rl, ....>
 
-Admin -> Peer_CLI ++: lifecycle chaincode\n createenclave (CC_Id, Attestation_Params)
+Admin -> Peer_CLI ++: lifecycle chaincode\n createenclave (CC_Id, Attestation_Params, [Peer_Endpoint])
```

Comment:
why is peer endpoint in `[]`? is it a array and can contain multiple endpoints? 

Also, I though the endpoint is "for now" a simple string, which also makes it easy to handle in the cli scripts? Below the endpoint is defined using some [fabric protos](https://github.com/hyperledger/fabric-protos/blob/master/discovery/protocol.proto#L232).

Reply:
this was meant to denote an optional parameter. But i guess i'll just drop that as initially we probably always need it explicitly from admin ...

Reply:
> why is peer endpoint in `[]`? is it a array and can contain multiple endpoints?

this was meant to denote an optional parameter. But i guess i'll just drop that as initially we probably always need it explicitly from admin ...

> Also, I though the endpoint is "for now" a simple string, which also makes it easy to handle in the cli scripts? Below the endpoint is defined using some [fabric protos](https://github.com/hyperledger/fabric-protos/blob/master/discovery/protocol.proto#L232).

We didn't talk about format and it seemed natural to use the fabric definition.   as long as we have an `initEnclave` message then i don't think that using a string naturally makes it fundamentally?